### PR TITLE
update windows guest telnet idle timeout and maxconn

### PR DESCRIPTION
**Pull request by xutian**
Signed-off-by: Xu Tian xutian@redhat.com

### CODE REVIEW

**Comment by xutian** [comment]
ID: 1377924

**Comment by xutian** [comment]
@ngu-niny can help you review this PR? thanks.

File: shared/unattended/win7-32-autounattend.xml [hunk 1]
```diff
@@ -169,53 +169,61 @@
...
 				</SynchronousCommand>
 				<SynchronousCommand wcm:action="add">
-					<CommandLine>%WINDIR%\System32\cmd /c netsh interface ip set address "Local Area Connection" dhcp</CommandLine>
+					<CommandLine>%WINDIR%\System32\cmd /c bcdedit /set {current} bootstatuspolicy ignoreallfailures</CommandLine>
 					<Order>11</Order>
 				</SynchronousCommand>
 				<SynchronousCommand wcm:action="add">
-					<CommandLine>%WINDIR%\System32\cmd /c KVM_TEST_VIRTIO_BALLOON_INSTALLER</CommandLine>
+					<CommandLine>%WINDIR%\System32\cmd /c E:\setuprss.bat</CommandLine>
 					<Order>12</Order>
```
Comment (ngu-niny) [inline shared/unattended/win7-32-autounattend.xml]: 2 'E:\setuprss.bat' in the xml file, and the same for file 'win8-32-autounattend.xml'; and there are also the same problem in the original xml files, not sure if it's wrong.

File: shared/unattended/win8-64-autounattend_ovmf.xml [hunk 2]
```diff
@@ -179,53 +179,61 @@
...
+					<CommandLine>%WINDIR%\System32\cmd /c E:\software_install_64.bat</CommandLine>
+					<Order>18</Order>
+				</SynchronousCommand>
+				<SynchronousCommand wcm:action="add">
+					<CommandLine>%WINDIR%\System32\cmd /c wmic datafile where "filename='finish' and extension='bat'" call copy "c:\\finish.bat" &amp;&amp; c:\finish.bat PROCESS_CHECK</CommandLine>
+					<Order>19</Order>
+				</SynchronousCommand>
 			</FirstLogonCommands>
 		</component>
 	</settings>
```
Comment (ngu-niny) [inline shared/unattended/win8-64-autounattend_ovmf.xml]: No change for win10 xml files

Comment (xutian) [inline shared/unattended/win8-64-autounattend_ovmf.xml]: no telnet server for windows, so this fix not impact win10

File: shared/unattended/win2008-32-autounattend.xml [hunk 3]
```diff
@@ -172,45 +172,53 @@
 					<Order>5</Order>
 				</SynchronousCommand>
 				<SynchronousCommand wcm:action="add">
-					<CommandLine>%WINDIR%\System32\cmd /c net start telnet</CommandLine>
+					<CommandLine>%WINDIR%\System32\cmd /c tlntadmn config maxconn=30</CommandLine>
```
Comment (CongLi) [inline shared/unattended/win2008-32-autounattend.xml]: @xutian 

Just curious, what's the previous maxconn?

Comment (xutian) [inline shared/unattended/win2008-32-autounattend.xml]: defaults values for windows guest is 8

**Comment by xutian** [comment]
@CongLi  what's your test results?

**Comment by CongLi** [comment]
Installation and general tests (single_driver_install) are fine.
But for that bug, the failed test case used "telnet' for the testing, but in my test is nc.

So I need check with this problem.

File: shared/unattended/win7-32-autounattend.xml [hunk 4]
```diff
@@ -169,53 +169,61 @@
 					<Order>5</Order>
 				</SynchronousCommand>
 				<SynchronousCommand wcm:action="add">
-					<CommandLine>%WINDIR%\System32\cmd /c net start telnet</CommandLine>
+					<CommandLine>%WINDIR%\System32\cmd /c tlntadmn config maxconn=30</CommandLine>
```
Comment (suqinhuang) [inline shared/unattended/win7-32-autounattend.xml]: is there any params that I check check in guest after instllation,
as I plan to update the num manually before, but didn't find the param

Comment (xutian) [inline shared/unattended/win7-32-autounattend.xml]: use 'tlntadmn' to list telnet server configurations

**Comment by CongLi** [comment]
Sorry that I forgot to update.
Test WIn8, the results are fine.

Acked-by: Cong Li <coli@redhat.com>

**Comment by xutian** [comment]
thanks @CongLi  test results. and Internal has used this patch long time, so merged backup to upstream.

**Pull request merged**